cursorless-talon/src/positional_target.py

```diff
@@ -0,0 +1,53 @@
+from talon import Context, Module
+
+from .modifiers.position import construct_positional_modifier
+
+mod = Module()
+ctx = Context()
+
+mod.list(
+    "cursorless_positional_connective",
+    desc='Connectives used to separate source and destination targets that indicate position, eg "before" or "after"',
+)
+
+ctx.lists["self.cursorless_positional_connective"] = {
+    "before": "before",
+    "after": "after",
+}
```

Comment:
I moved these positional connectives here, because in my mind it's a bit of a hack that we allow them as modifiers; to me they are more properly considered to be connectives.  Open to tweaking if you think we should migrate them back to the other file; shouldn't be too big of a change

Reply:
If we are going to remove before and after I definitely think that we should add positions to csv file so user can customize them because start of an end of is to verbose. I also think that we should disable chuck after since if we're going to do this puristicly we're going to do it all the way.